### PR TITLE
Add support for negative prompt

### PR DESCRIPTION
**Pull request by vshashkov**
# What is this?
Added support for new param: `negative_prompt` (extracted this part from the [prev pr](https://github.com/replicate/cog-stable-diffusion/pull/43)).

# Why?
We have a lot of requests in the issues for negative prompt support, that's why I've created this PR.

# How did you test to ensure no regressions?
I've used [brev console](https://console.brev.dev/) for testing it.
Screenshot 2022-11-04 at 21 31 39

# Testing results:
Below are my testing results from the [previous pr](https://github.com/replicate/cog-stable-diffusion/pull/43):

**prompt:**
_Ragdoll cat king wearing a golden crown, intricate, elegant, highly detailed, centered, digital painting, artstation, concept art, smooth, sharp focus, illustration, artgerm, Tomasz Alen Kopera, Peter Mohrbacher, donato giancola, Joseph Christian Leyendecker, WLOP, Boris Vallejo_

**seed:** _789_

Screenshot 2022-11-04 at 21 25 25

and with **negative prompt** _yellow color, cat, nose_ with `K_EULER_ANCESTRAL`:
Screenshot 2022-11-04 at 21 35 55

### CODE REVIEW

**Comment by zeke** [comment]
Nice! Adding in @chenxwh for review. 🙏🏼

**Review by chenxwh — APPROVED** [review]

**Comment by vshashkov** [comment]
@bfirsh @zeke can we merge this?

**Comment by zeke** [comment]
@vshashkov I'm checking with the team about our process for merging PRs on this repo and publishing the changes to Replicate. We should be able to get this shipped soon. Thanks for your patience.

File: predict.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ def setup(self):
     def predict(
         self,
         prompt: str = Input(description="Input prompt", default=""),
+        negative_prompt: str = Input(description="Input negative prompt", default=""),
```
Comment (chenxwh) [inline predict.py:54]: suggestion: change to `negative_prompt: str = Input(description="Input negative prompt", default=None)` since it is optional

**Comment by therealadityashankar** [comment]
this is amazing!, I'd really love this for a product I'm trying to build

File: predict.py (hunk 1 above)
Comment (vshashkov) [reply predict.py]: Thank you, @chenxwh! Updated!

**Comment by chenxwh** [comment]
@vshashkov before this is merged and pushed to the main demo, you can try these updates here https://replicate.com/cjwbw/stable-diffusion

**Comment by zeke** [comment]
https://replicate.com/stability-ai/stable-diffusion now has support for negative prompts!

Screen Shot 2022-11-28 at 8 20 51 AM

@chenxwh can this PR be merged as-is, or did you have to make some other changes?

**Comment by chenxwh** [comment]
I pushed the version with more changes e.g. more schedulers, but for the negative examples this PR is enough

**Comment by zeke** [comment]
@chenxwh so does the main branch of this repo currently correspond to what's published at https://replicate.com/stability-ai/stable-diffusion/versions/27b93a2413e7f36cd83da926f3656280b2931564ff050bf9575f1fdf9bcd7478 ?

**Pull request merged**